src/pages/user/review/[username].tsx

```diff
@@ -1,57 +1,223 @@
 import { BackButton } from "@/components/backbutton";
 import { Button } from "@/components/button";
-import Image from "next/image";
+import { useState,useEffect } from "react";
+import { UserProfile } from "@/services/user.service";
+import { getTrainerProfile, TrainerProfile } from "@/services/trainer.service";
+import { useRouter } from "next/router";
+import { getTrainerReviews, ReviewDetail } from "@/services/trainer.service";
+import {StarIcon} from "@heroicons/react/24/outline";
 
 const Review = () => {
-  const ReviewBox = () => {
+  const [showModal, setShowModal] = useState(false);
+
+  const [trainerProfile, setTrainerProfile] = useState<UserProfile>({
+    username: "user",
+    firstname: "firstname",
+    lastname: "lastname",
+    birthdate: new Date(),
+    citizenId: "none",
+    gender: "none",
+    phoneNumber: "none",
+    address: "none",
+    usertype: "none",
+    lat: 0,
+    lng: 0,
+  });
+
+  
+
+  const [trainerReviews, setTrainerReviews] = useState<ReviewDetail[]>([]);
+
+  const router = useRouter();
+  const { username } = router.query;
+  const [loading, setLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+  if (typeof username == "string") {
+    getTrainerProfile(username)
+      .then((res1) => {
+        setLoading(true);
+        setTrainerProfile(res1.userProfile);
+        setLoading(false);
+      })
+      .catch(() => router.back())
+    getTrainerReviews(username)
+      .then((res2) => {
+        setLoading(true);
+        setTrainerReviews(res2.reviewList);
+        setLoading(false);
+      })
+    .catch(() => router.back())
+  } else {
+    router.back();
+  }
+}, [username]);
+
+  const Image = () => {
     return (
-      <div className="h-[20%] w-[80%] bg-white mb-[5%] border-2 border-gray rounded-3xl items-center justify-between drop-shadow-lg hover:bg-gray-light"></div>
+      <div className="max-h-min overflow-hidden mt-6 md:mt-10 flex justify-center">
+        <img
```

Comment:
I already remove this and use `<Image ....... />` from next/Image, please check main branch and fix this.